Can't compile docker container

I'm getting this when running `docker build .` for the dockerfile. 

```
In file included from zimwriterfs.cpp:37:0:
mimetypecounter.h:50:8: error: ‘map’ in namespace ‘std’ does not name a template type
   std::map<std::string, unsigned int> counters;
        ^
Makefile:532: recipe for target 'zimwriterfs-zimwriterfs.o' failed
make: *** [zimwriterfs-zimwriterfs.o] Error 1
The command '/bin/sh -c cd zimwriterfs && make all install' returned a non-zero code: 2

```

This is on Ubuntu 14.04.

--- a/mimetypecounter.h
+++ b/mimetypecounter.h
@@ -23,6 +23,7 @@
 
 #include "article.h"
 #include "articlesource.h"
+#include <map>
 
 class MimetypeCounter;
 

